api4/post.go

```diff
@@ -67,7 +68,20 @@ func createPost(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	c.App.SetStatusOnline(c.App.Session.UserId, false)
+	setOnline := r.URL.Query().Get("set_online")
+	setOnlineBool := true // By default, always set online.
+	var err2 error
+	if setOnline != "" {
+		setOnlineBool, err2 = strconv.ParseBool(setOnline)
+		if err2 != nil {
+			mlog.Error(err2.Error())
```

Comment:
I don't think we should log an error here. A warning is enough, as the system still works fine. There should also be an explanation, why the errors is shown in the logs.
```suggestion
			mlog.Warn("Failed to parse set_online url query parameter from createPost request", mlog.Err(err2))
```